### ISSUE TITLE
document the encoding of connection to wikidata in the encoding guidelines

### ISSUE THREAD

**Issue by ingoboerner** [dracor-org/dracor-schema#53]
We have a peculiar encoding of linking the "work" to the respective entity on wikidata. It should be explained in the encoding guidelines.

**Comment by lb42** [comment]
Yes please. How should this be done? And what about works for which wikidata is missing?

**Comment by cmil** [comment]
We may change how it's done in a near future schema version, see #116.
